lib/plugins/inspec-compliance/lib/inspec-compliance/cli.rb

```diff
@@ -72,10 +72,9 @@ def profiles
       desc "exec PROFILE", "executes a #{COMPLIANCE_PRODUCT_NAME} profile"
       exec_options
       def exec(*tests)
-        config = InspecPlugins::Compliance::Configuration.new
-        return unless loggedin(config)
-
```

Comment:
Please don't remove newlines (aka paragraph separators) that aid in readability and flow of the code.